Makefile

```diff
@@ -133,14 +128,14 @@ unit-test: ## Runs unit test without e2e
 e2e: ## Runs e2e tests. Requires GITHUB_AUTH_TOKEN env var to be set to GitHub personal access token
 e2e: build-scorecard check-env | $(GINKGO)
 	# Run e2e tests. GITHUB_AUTH_TOKEN with personal access token must be exported to run this
-	ginkgo --skip="E2E TEST:executable" -p -v -cover  ./...
+	$(GINKGO) --skip="E2E TEST:executable" -p -v -cover  ./...
 
 ci-e2e: ## Runs CI e2e tests. Requires GITHUB_AUTH_TOKEN env var to be set to GitHub personal access token
-ci-e2e: build-scorecard check-env e2e-cron | $(GINKGO)
+ci-e2e: build-scorecard check-env e2e-cron install| $(GINKGO)
```

Comment:
Specifying `install` as dependency will always install all dependencies when we run `make ci-e2e`. Instead, we can have `| $(GINKGO)` and provide a rule for `$(GINKGO)` as:

```
$(GINKGO): install
```

That way, it'll only run the install command if `$(GINKGO)` file doesn't exist. Wdyt?

Reply:
Yes agreed. Thanks!